config/servicemappings/vertexai.yaml

```diff
@@ -61,3 +61,54 @@ spec:
             kind: KMSCryptoKey
             version: v1beta1
             group: kms.cnrm.cloud.google.com
+    - name: google_vertex_ai_endpoint
+      kind: VertexAIEndpoint
+      idTemplate: "projects/{{project}}/locations/{{location}}/endpoints/{{name}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: true
+      v1alpha1ToV1beta1: true
+      storageVersion: v1alpha1
```

Comment:
Nit: maybe should be v1beta1?  We should figure out how we eventually stop shipping v1alpha1 CRD versions!

Reply:
When we promote a resource from alpha to beta, we set `v1alpha1ToV1beta1` to `true` and `storageVersion` to `v1alpha1` according to the [New Resource Guide](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#add-the-resource-config-to-service-mappings).
@maqiuyujoyce Do we have a process of eventually stop shipping v1alpha1 CRDs?

Reply:
We do have a proposal but further work is needed to implement it. Will follow up offline about it.